app/src/main/java/com/github/h3lp3rs/h3lp/messaging/Conversation.kt

```diff
@@ -179,12 +150,98 @@ class Conversation(
         database.delete(conversationId)
     }
 
+
+    /**
+     * Load the cache if exist
+     * @warning should be called in onCreate method of the activity
+     */
+    fun loadCache() {
```

Comment:
Might be best to give this method a name related to the messaging feature, as one might think loadCache would load everything ;)